packages/L10n/stories/examples/withLocales/FakeAppWithLocales.js

```diff
@@ -1,17 +1,17 @@
 import React from "react";
 import { Story } from "storybook/assets/styles/common.styles";
-import FormElement, { Label } from "@paprika/form-element";
+import FormElement from "@paprika/form-element";
 import L10n from "../../../src/L10n";
 import Locales from "./Locales";
 import Greeting from "./Greeting";
 
-export default function FakeAppWithLocales(props) {
+export default function FakeAppWithLocales() {
   return (
     <Story>
-      <L10n locale={props.locale} locales={Locales}>
+      <L10n locale="en" locales={Locales}>
         <Greeting />
```

Comment:
Not really part of this task, but it might make more sense to define `<Greeting>` inline (even if it means the code will be less DRY) for clarity when someone looks at this story source code.

Reply:
It doesn't work at the moment with it inline

Reply:
I guess "inline" wasn't the right way to explain it.  I mean just moving the `Greeting` component right into this same file, so _this_ line would actually remain the same.  But really not required, it's certainly out of scope.